.ruby-version

```diff
@@ -0,0 +1 @@
+2.5.1
```

Comment:
do we need this?

i think in `.travis.yml` we can just specify `rvm: 2.5.1`

(would rather keep this in one place)

Reply:
You can use either.  My reasoning: Ensure the ruby version is locked/consistent whether running on travis or locally.  

Which do you prefer?

Reply:
let's go with specifying in only the `.travis.yml` file for now. if needed, we can revisit.